src/Symfony/Component/Asset/VersionStrategy/JsonManifestVersionStrategy.php

```diff
@@ -24,15 +27,20 @@
  */
 class JsonManifestVersionStrategy implements VersionStrategyInterface
 {
+    use ManifestAlternativesTrait;
+
     private $manifestPath;
     private $manifestData;
+    private $strict;
```

Comment:
Can you please add bool phpdoc?

Reply:
The constructor param has already the phpdoc.